### PR TITLE
Get File Sample From Path - Generic v3 - added AcquiredFile output

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
Related to: https://jira-dc.paloaltonetworks.com/browse/CIAC-8220

## Description
Added the AcquiredFile output, which can aid in the distinction between the existing incident files and the ones retrieved using PS Remote.

### CODE REVIEW

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_14.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Get File Sample From Path - Generic V3
+
+- Added the AcquiredFile output, which will help to distinguish the existing files from the file that has just been retrieved.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_14.md:6]:
```suggestion
Added the AcquiredFile output, which will help to distinguish the existing files from the file that has just been retrieved.
```

**Comment by ShirleyDenkberg** [comment]
@michalgold @AdiPeret

**Review by ssokolovich — COMMENTED** [review]
Just to be sure, which sub-playbook provided this key `AcquiredFile`?

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_From_Path_-_Generic_V3.yml [hunk 2]
```diff
@@ -271,6 +285,9 @@ outputs:
 - contextPath: File.SSDeep
   description: File SSDeep.
   type: string
+- contextPath: AcquiredFile
+  description: The acquired file details.
+  type: string
```
[the remark this answers is not in the document]
Comment (idovandijk) [reply Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_From_Path_-_Generic_V3.yml:290]: That is from the PS Remote subplaybook
image

it seems like it selected String by default, but it's actually not a string. I'll fix.

[the remark this answers is not in the document]
Comment (ssokolovich) [reply Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_From_Path_-_Generic_V3.yml:290]: So maybe we should make it unknown then? cause string can be problematic when you use transformers - it will filter the options (unless you click on expand at the top).
TL;DR -  I think unknown in this case can be much better.

**Review by ssokolovich — APPROVED** [review]

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_From_Path_-_Generic_V3.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (idovandijk) [reply Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_From_Path_-_Generic_V3.yml]: Yeah fixed it
